e2e/test/scenarios/admin/datamodel/datamodel.cy.spec.js

```diff
@@ -420,6 +420,40 @@ describe("scenarios > admin > datamodel > metadata", () => {
     cy.signInAsAdmin();
   });
 
+  it("should remap FK display value from field ", () => {
+    cy.wrap(
+      `/admin/datamodel/database/${SAMPLE_DB_ID}/schema/${SAMPLE_DB_SCHEMA_ID}/table/${ORDERS_ID}/field/${ORDERS.PRODUCT_ID}/general`,
+    ).as("ORDERS_PRODUCT_ID_URL");
+
+    visitAlias("@ORDERS_PRODUCT_ID_URL");
+
+    cy.findByPlaceholderText("PRODUCT_ID")
+      .clear()
+      .type("Remapped Product ID")
+      .realPress("Tab");
+
+    openOrdersTable({ limit: 5 });
```

Comment:
Let's wait for a `PUT` call before navigating

Reply:
[await for PUT](https://github.com/metabase/metabase/pull/45055/commits/133c43be1fa98c7e132cb7f3a9dd17cb44faa6bf)